kpi/views/v1/import_task.py

```diff
@@ -59,6 +62,53 @@ def create(self, request, *args, **kwargs):
                 'importtask-detail',
                 kwargs={'uid': import_task.uid},
                 request=request),
-            'status': ImportTask.PROCESSING
+            'status': ImportTask.PROCESSING,
+            'default_project_name': default_project_name
         }, status.HTTP_201_CREATED)
 
+    @staticmethod
+    def _get_filenames(
+        request: Request,
+    ) -> Union[Tuple[str, str], Tuple[None, None]]:
+        """
+        Return a tuple of the submitted file or url-linked file's name with
+        and without its extension
+
+        Args:
+            request (Request): DRF request object
+
+        Returns:
+            Tuple[str, str] or Tuple[None, None]: A tuple containing first the
+            filename with its extension and secondly the filename without its
+            extension. If there is no filename, a tuple of None values is
+            returned
+        """
+
+        XLS_FILE_PATTERN = '((.*?)\.xlsx?)'
```

Comment:
`os.path.splitext()` is pretty good at this:
```
In [8]: os.path.splitext('ab.cd efg h. ij .k lmnop.xls')                                                                                                                                       
Out[8]: ('ab.cd efg h. ij .k lmnop', '.xls')
```

Reply:
Taking the context of the method pattern and the fact that the `rfc6266` seems to be broken, I'm inclined to leave it as a regex — especially considering that we also only accept `xls` and `xlsx` file extensions — `os.path.splitext()` doesn't care. @noliveleger are you happy with this?